pages/players/[playerID].tsx

```diff
@@ -173,18 +190,29 @@ export const getServerSideProps: GetServerSideProps<any, { playerID: string }> =
   params,
   query,
   res,
+  req,
 }) => {
   const { playerID } = params!;
   const { view } = query;
 
   const viewPlayerMatches = view === "recentMatches";
+  const viewStandings = view === "standings";
 
   let playerData = null;
   let playerMatchesData = null;
   let error = null;
 
+  const prevPage = req.headers.referer;
+  const prevPlayerId = prevPage?.split("/")[4];
```

Comment:
Hm wow, this works pretty well 👍 nice job 

This is the only thing which we should change, splitting like this is always dangerous. 
I would probably use this regex: 
```
/.+players\/(\d+).+/
```
And the playerId should be in group1 

https://regex101.com/r/HLgvQc/1